### ISSUE TITLE
rigel miner on kryptex

### ISSUE THREAD

**Issue by dubbs1980** [RainbowMiner/RainbowMiner#2635]
getting login errors on kryptex with rigel miner

**Comment by RainbowMiner** [comment]
For which algorithm did you get the login error? Or are these all kinds of?

**Comment by dubbs1980** [comment]
pyi and kls. it looks like there is no autoexchange for these on the pool, so that would explain the login error. it was my mistake.

**Comment by RainbowMiner** [comment]
Yes, I added one commit for Kryptex today to detect auto-exchange coins: commit 6ec65cbc742540971d9521e63880931d4db90f92
Plus, I have added detection for eMail accounts to BzMiner (which cannot handle an eMail as pool user): commit b3b938ca07d19110d1b084917419fd233ab151a9
